account/account.go

```diff
@@ -533,6 +533,27 @@ func (account *Account) WaitForTransactionReceipt(ctx context.Context, transacti
 	}
 }
 
+// AddTransaction calls dynamically one of Add<type>Transaction methods
+//
+// Parameters:
+// - ctx: the context.Context object for the transaction.
+// - txn: this Broadcast Transaction to be called.
+// Returns:
+// - interface{} returns required Broadcast txn return value.
+// - error: an error if any.
+func (account *Account) AddTransaction(ctx context.Context, txn rpc.BroadcastTxn) (interface{}, error) {
+	switch tx := txn.(type) {
+	case rpc.BroadcastInvokeTxnType:
+		return account.AddInvokeTransaction(ctx, tx)
```

Comment:
Let's remove these three methods from the "account" package. Call them directly from the "account.provider" instead. You can also remove [this ](https://github.com/NethermindEth/starknet.go/blob/main/account/account.go#L43)line

Reply:
Done